napalm_ios/ios.py

```diff
@@ -1223,7 +1225,7 @@ def get_interfaces_counters(self):
             for line in interface_str.splitlines():
                 if 'packets input' in line:
                     # '0 packets input, 0 bytes, 0 no buffer'
-                    match = re.search(r"(\d+) packets input.*(\d+) bytes", line)
+                    match = re.search(r"(\d+) packets input.* (\d+) bytes", line)
```

Comment:
@ktbyers:

Without space it did not seem to catch all of the bytes:
[https://regex101.com/r/cQBAt5/1
](https://regex101.com/r/cQBAt5/1
)
With the space it looks like it works:
[https://regex101.com/r/TZIOf3/1](https://regex101.com/r/TZIOf3/1)